### ISSUE TITLE
Combustion Charge breaks bedrock

### ISSUE THREAD

**Issue by demisys** [Safrodev/Archon#15]
Right clicking with a Combustion Charge on a bedrock block breaks the one bedrock block you're looking at (and not its neighbours).
```
public ActionResult useOnBlock(ItemUsageContext context) {
        PlayerEntity player = context.getPlayer();
        World world = context.getWorld();
        for (BlockPos pos : ArchonUtil.getPosArea(context.getWorld(), context.getBlockPos(), context.getPlayer(), 3, true)) {
            if (canBreak(player, world, context.getBlockPos(), pos)) {
```

Seems to be the `placeDummyBlocks` parameter in the `ArchonUtil.getPosArea` call. Looks like it sets the block you're looking at to Dirt and then to Air?

**Comment by Safrodev** [comment]
Ah you are right, I’ll have to put a check in the getPosArea method. It needs some changing anyway as the code is pretty outdated.

**Comment by Safrodev** [comment]
Should be fixed in v0.4.0 sorry for the delay
